src/components/Market/MarketFilter/index.tsx

```diff
@@ -0,0 +1,313 @@
+          >
+            <Typography
+              sx={{
+                fontSize: '1rem',
+                fontWeight: 700,
+                color: theme.palette.common.black,
+              }}
+            >
+              Filters
+            </Typography>
+            <Button
+              startIcon={<CloseOutlinedIcon />}
+              sx={{
+                color: theme.palette.common.black,
+                display: 'flex',
+                alignItems: 'center',
+              }}
+              onClick={() => clearFilters()}
+            >
+              Clear all
+            </Button>
+          </Stack>
+          <Stack direction='column' gap='1rem' sx={{ my: '1rem' }}>
+            <FilterItem label='Start date'>
+              <DateInput value={startDate} onChange={(v) => setStartDate(v)} />
+            </FilterItem>
+            <FilterItem label='End date'>
+              <DateInput
+                value={endDate}
+                onChange={(v) => setEndDate(v)}
+                minDate={startDate === null ? undefined : startDate}
+              />
+            </FilterItem>
+            <FilterItem label='Core Occupancy'>
+              <Select
+                value={occupancyIndex}
+                onChange={(e) => setOccupancyIndex(e.target.value as number)}
+                sx={{
+                  '.MuiSelect-select': {
+                    background: theme.palette.common.white,
+                  },
+                  '.MuiOutlinedInput-notchedOutline': {
+                    borderRadius: '0.5rem',
+                  },
+                  '.MuiOutlinedInput-input': {
+                    py: '0.75rem',
+                  },
+                }}
+              >
+                {coreOccupancyOptions.map(({ label }, index) => (
+                  <MenuItem key={label} value={index}>
+                    {label}
+                  </MenuItem>
+                ))}
+              </Select>
+            </FilterItem>
+            <FilterItem label='Region duration at least'>
+              <Select
+                value={durationIndex}
+                onChange={(e) => setDurationIndex(e.target.value as number)}
+                sx={{
+                  '.MuiSelect-select': {
+                    background: theme.palette.common.white,
+                  },
+                  '.MuiOutlinedInput-notchedOutline': {
+                    borderRadius: '0.5rem',
+                  },
+                  '.MuiOutlinedInput-input': {
+                    py: '0.75rem',
+                  },
+                }}
+              >
+                {durationOptions.map(({ label }, index) => (
+                  <MenuItem key={label} value={index}>
+                    {label}
+                  </MenuItem>
+                ))}
+              </Select>
+            </FilterItem>
+            <FilterItem label='Price range'>
+              <Slider
+                value={priceRange}
+                max={150}
```

Comment:
We should make this dynamic since it is possible that regions will cost more than 150 ROC or 150 KSM.

Reply:
Should probably be set to the most expensive region price